tools/js-tools/eslintrc/base.js

```diff
@@ -168,5 +168,6 @@ module.exports = {
 
 		// Disabled pending #16099.
 		'inclusive-language/use-inclusive-words': 0,
+		'no-use-before-define': 'off',
```

Comment:
If this addition is needed, move it up so it doesn't seem attached to the comment on line 169.

Reply:
Done. Thanks